config/routes.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'party/create'
```

Comment:
@manojpanta I think these two might have automatically generated from the scaffolded Party. Do we need them?